conanbuildinfo.cmake

```diff
@@ -0,0 +1,184 @@
+set(CONAN_INCLUDE_DIRS  ${CONAN_INCLUDE_DIRS})
+set(CONAN_LIB_DIRS  ${CONAN_LIB_DIRS})
+set(CONAN_BIN_DIRS  ${CONAN_BIN_DIRS})
+set(CONAN_LIBS  ${CONAN_LIBS})
```

Comment:
Yeah.

On Mon, Apr 18, 2016 at 2:26 PM Jack Poulson <notifications@github.com>
wrote:

> In conanbuildinfo.cmake
> <https://github.com/scibuilder/metis/pull/1#discussion_r60137195>:
>
> > @@ -0,0 +1,184 @@
> > +set(CONAN_INCLUDE_DIRS  ${CONAN_INCLUDE_DIRS})
> > +set(CONAN_LIB_DIRS  ${CONAN_LIB_DIRS})
> > +set(CONAN_BIN_DIRS  ${CONAN_BIN_DIRS})
> > +set(CONAN_LIBS  ${CONAN_LIBS})
>
> Are they no-ops?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/scibuilder/metis/pull/1/files/e75ca6e9db76dcd9d32167cf669e95276b7620dd#r60137195>
>